src/@types/spaces.ts

```diff
@@ -15,26 +15,46 @@ limitations under the License.
 */
 
 import { IPublicRoomsChunkRoom } from "../client";
+import { RoomType } from "./event";
+import { IStrippedState } from "../sync-accumulator";
 
 // Types relating to Rooms of type `m.space` and related APIs
 
 /* eslint-disable camelcase */
+/** @deprecated */
 export interface ISpaceSummaryRoom extends IPublicRoomsChunkRoom {
     num_refs: number;
     room_type: string;
 }
 
+/** @deprecated */
 export interface ISpaceSummaryEvent {
```

Comment:
I'm surprised we don't have a generic event interface yet

Reply:
We do, this predates it, keeping the identifier until deprecation expires and this gets canned